tests/bears/requirements/DistributionRequirementTest.py

```diff
@@ -8,16 +8,16 @@
 
```

Comment:
helping you out: add this decorator to the class:

``@unittest.skipIf(platform.system() == 'Windows', "This test cannot be run on Windows.")``

and import platform anyway i guess ;d